suite-native/biometrics/src/components/BiometricsIcon.tsx

```diff
@@ -0,0 +1,55 @@
+import { Platform } from 'react-native';
+
+import { IconName, Icon } from '@suite-common/icons';
+import { Box } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+const ICON_SIZE_DEFAULT = 64;
+const ICON_WRAPPER_PADDING = 12;
+
+const iconWrapperStyle = prepareNativeStyle(
+    (utils, { wrapperSize, showShadow }: { wrapperSize: number; showShadow: boolean }) => ({
+        padding: ICON_WRAPPER_PADDING,
+        borderRadius: utils.borders.radii.round,
+        backgroundColor: utils.colors.backgroundSurfaceElevation2,
+        color: utils.colors.iconPrimaryDefault,
+        width: wrapperSize,
+        height: wrapperSize,
+
+        extend: {
+            condition: showShadow,
+            style: {
+                shadowColor: utils.boxShadows.small.shadowColor,
+                shadowOffset: utils.boxShadows.small.shadowOffset,
+                shadowRadius: utils.boxShadows.small.shadowRadius,
+                elevation: utils.boxShadows.small.elevation,
+                shadowOpacity: utils.boxShadows.small.shadowOpacity,
+            },
+        },
+    }),
+);
+
+type BiometricsIconProps = {
+    iconSize?: number;
```

Comment:
For now, I think `IconSize` (32) is the only value used to send down to this component as size. We could try to keep it tied to the theme sizes and extend if needed. Not a big deal though..

```suggestion
    iconSize?: IconSize;
```

Reply:
Default size is 64, that one is not in icon sizes. And I derive wrapper size from icon size